blocks/extra-large-promo-block/features/extra-large-promo/default.jsx

```diff
@@ -45,7 +46,12 @@ const ExtraLargePromo = ({ customFields }) => {
       : null,
   }) || null;
 
-  const imageConfig = customFields.imageOverrideURL ? 'resize-image-api' : null;
+  let imageConfig = null;
+  if (customFields.imageOverrideURL && customFields.lazyLoad) {
```

Comment:
should this be isAdmin check as well? I feel like we've done that before

Reply:
No, see comment below